### PR TITLE
add: `tdrop-git`

### PR DESCRIPTION
**Pull request by thomascrha**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [x] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

File: packages/tdrop-git/tdrop-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,17 @@
+maintainer="Thomas Crha <thomas@9bitbyte.com>"
+
+name="tdrop-git"
+gives="tdrop"
+url="https://github.com/noctuid/tdrop.git"
+pkgver="0.5.0"
+homepage='https://github.com/noctuid/tdrop'
+pkgdesc="A Glorified WM-Independent Dropdown Creator"
+makedepends=("cmake")
+depends=("xdotool", "bash", "gawk", "procps-ng")
```
Comment (Elsie19) [inline packages/tdrop-git/tdrop-git.pacscript:10]: Remove the commas.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by thomascrha** [comment]
Applogies for the force push - LazyGit just started doing it before I could respond. Anyway thats the commas removed.

**Comment by Elsie19** [comment]
Great! Can you pull from master so the CI will start working?

**Comment by thomascrha** [comment]
Is that what you wanted ?

**Comment by Elsie19** [comment]
I meant from the GitHub interface 👀

**Comment by thomascrha** [comment]
Do you want me throw this out and start again ? This branches history is now pretty messed up. I can squash it all down ?

**Comment by thomascrha** [comment]
Closing this due to some git errors on my part - will open another pr to keep things clean.

**Pull request closed without merging**